modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
```

Comment:
Would "Annotating namespaces and pods for workload partitioning" be an alternative?

Annotating resources for workload partitioning
Adding annotations for workload partitioning

If this is a conceptual module, then maybe "Required annotations for workload partitioning"--and that might be fine because there isn't a `.Procedure` block title or steps like I'd expect to find, according to the mod docs guidelines for creating a [procedure module](https://redhat-documentation.github.io/modular-docs/#creating-procedure-modules).